GothicVRProject/GothicVR
# Feature/sky improvements

Sky v2 - rain implemented, new texture per day, fog colors from INI

# To test
1. Set time to 16:25 and see if it rains at 16:30 to 17:30
2. Set time to 23:59 and see if the proper textures appear during the day (gray clouds)

# Testing
